### PR TITLE
Fix white stroke color not working

### PR DESCRIPTION
**Pull request by bryant1410**

### CODE REVIEW

File: library/src/main/java/com/dd/CircularProgressButton.java [hunk 1]
```diff
@@ -129,11 +130,11 @@ private void initCompleteStateDrawable() {
         int colorPressed = getPressedColor(mCompleteColorState);
 
         StrokeGradientDrawable drawablePressed = createDrawable(colorPressed);
-        if (mStrokeColor != -1) {
+        if (mStrokeColor != COLOR_TRANSPARENT) {
```
Comment (mirland) [inline library/src/main/java/com/dd/CircularProgressButton.java]: :+1:

File: library/src/main/java/com/dd/CircularProgressButton.java [hunk 2]
```diff
@@ -19,6 +19,7 @@
 import com.dd.circular.progress.button.R;
 
 public class CircularProgressButton extends Button {
+    private static final int COLOR_TRANSPARENT = 0;
```
Comment (mirland) [inline library/src/main/java/com/dd/CircularProgressButton.java]: Could you get it from an android class?

Comment (bryant1410) [inline library/src/main/java/com/dd/CircularProgressButton.java]: yes

**Comment by bryant1410** [comment]
Comments fixed

**Pull request merged**